WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ReprintShippingLabelViewController.swift

```diff
@@ -0,0 +1,257 @@
+private extension ReprintShippingLabelViewController {
+    func configure(_ cell: UITableViewCell, for row: Row) {
+        switch cell {
+        case let cell as BasicTableViewCell where row == .headerText:
+            configureHeaderText(cell: cell)
+        case let cell as TopLeftImageTableViewCell where row == .infoText:
+            configureInfoText(cell: cell)
+        case let cell as SettingTitleAndValueTableViewCell where row == .paperSize:
+            configurePaperSize(cell: cell)
+        case let cell as SpacerTableViewCell where row == .spacer:
+            configureSpacer(cell: cell)
+        case let cell as TopLeftImageTableViewCell where row == .paperSizeOptions:
+            configurePaperSizeOptions(cell: cell)
+        case let cell as TopLeftImageTableViewCell where row == .printingInstructions:
+            configurePrintingInstructions(cell: cell)
+        default:
+            break
+        }
+    }
+
+    func configureHeaderText(cell: BasicTableViewCell) {
+        cell.textLabel?.text = Localization.headerText
+        cell.textLabel?.numberOfLines = 0
+        cell.textLabel?.applyBodyStyle()
+        cell.hideSeparator()
+    }
+
+    func configureInfoText(cell: TopLeftImageTableViewCell) {
+        cell.imageView?.image = .infoOutlineImage
+        cell.imageView?.tintColor = .systemColor(.secondaryLabel)
+        cell.textLabel?.text = Localization.infoText
+        cell.textLabel?.applyBodyStyle()
+        cell.textLabel?.textColor = .systemColor(.secondaryLabel)
+        cell.hideSeparator()
+    }
+
+    func configurePaperSize(cell: SettingTitleAndValueTableViewCell) {
+        cell.updateUI(title: Localization.paperSizeSelectorTitle, value: selectedPaperSize?.description)
+        cell.accessoryType = .disclosureIndicator
+    }
+
+    func configureSpacer(cell: SpacerTableViewCell) {
+        cell.height = Constants.verticalSpacingBetweenPaperSizeSelectorAndInfoLinks
+    }
+
+    func configurePaperSizeOptions(cell: TopLeftImageTableViewCell) {
+        cell.imageView?.image = .pagesFootnoteImage
+        cell.textLabel?.text = Localization.paperSizeOptionsButtonTitle
+        configureCommonStylesForInfoLinkCell(cell)
+    }
+
+    func configurePrintingInstructions(cell: TopLeftImageTableViewCell) {
+        cell.imageView?.image = .infoOutlineFootnoteImage
+        cell.textLabel?.text = Localization.printingInstructionsButtonTitle
+        configureCommonStylesForInfoLinkCell(cell)
+    }
+
+    func configureCommonStylesForInfoLinkCell(_ cell: TopLeftImageTableViewCell) {
+        cell.imageView?.tintColor = .systemColor(.secondaryLabel)
+        cell.textLabel?.applyFootnoteStyle()
+        cell.textLabel?.textColor = .systemColor(.secondaryLabel)
+        cell.hideSeparator()
+        cell.selectionStyle = .default
+    }
+}
+
+private extension ReprintShippingLabelViewController {
+    enum Constants {
+        static let cellValueTextColor = UIColor.systemColor(.secondaryLabel)
```

Comment:
⚠️ This seems to be unused.

```suggestion

```

Reply:
thanks for catching this! I wish Xcode has an unused code highlighter. Removed in https://github.com/woocommerce/woocommerce-ios/pull/3342/commits/341def7b4e3b1c75abc921c16ac423e0dd4945a6